### PR TITLE
LAYOUTS-616 Channel & locale condition types

### PR DESCRIPTION
**Pull request by RandyCupic**
This PR introduces two new condition types for Sylius channel and locale. Everything is covered with tests.

### CODE REVIEW

File: lib/Parameters/Form/Mapper/ChannelMapper.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    /**
+     * @return array<string, int>
+     */
+    private function getChannelOptions(): array
+    {
+        $channels = $this->channelRepository->findAll();
+        $channelList = [];
+
+        foreach ($channels as $channel) {
```
Comment (emodric) [inline lib/Parameters/Form/Mapper/ChannelMapper.php:43]: Probably needs an inline typehint for `$channel` to fix PHPStan errors.

**Comment by emodric** [comment]
Needs rebase on `master` (pull first, to fix two unrelated PHPStan issues)

File: lib/Parameters/Form/Mapper/ChannelMapper.php (hunk 1 above)
Comment (RandyCupic) [reply lib/Parameters/Form/Mapper/ChannelMapper.php:43]: @emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/4/commits/c830cbef2d3ad1e6f6d5235720acc4d0567f85e9

**Comment by RandyCupic** [comment]
@emodric Rebased to master.

**Pull request merged**